src/main/java/org/generationcp/middleware/service/impl/dataset/DatasetServiceImpl.java

```diff
@@ -158,6 +168,9 @@ public class DatasetServiceImpl implements DatasetService {
 		TermId.EXPERIMENT_DESIGN_FACTOR.getId());
 	private static final String SUM_OF_SAMPLES = "SUM_OF_SAMPLES";
 
+	private static final List<Integer> EXP_DESIGN_VARIABLES =
```

Comment:
I think you can just use the existing final AppConstants.EXP_DESIGN_VARIABLES:

` List<Integer> expDesignConstants = AppConstants.EXP_DESIGN_VARIABLES.getIntegerList()`

Reply:
AppConstants are in Commons which can't be accessed in MW